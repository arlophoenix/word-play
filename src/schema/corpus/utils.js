/* @flow */

// eslint-disable-next-line import/prefer-default-export
export function lexSort(letters: string) {
  return letters
    .toLowerCase()
    .split('')
    .sort()
    .join('');
}
