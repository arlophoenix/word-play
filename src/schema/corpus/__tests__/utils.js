/* eslint-env jest */

import { lexSort } from '../utils';

test('lexSort()', () => {
  expect(lexSort('Arlo')).toMatchSnapshot();
});
