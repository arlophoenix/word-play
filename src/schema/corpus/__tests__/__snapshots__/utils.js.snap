// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`lexSort() 1`] = `"alor"`;
